dnf-docker-test/features/steps/command_steps.py

```diff
@@ -34,6 +34,23 @@ def step_the_command_should_pass(ctx):
 def step_the_command_should_fail(ctx):
     ctx.assertion.assertNotEqual(ctx.cmd_result.returncode, 0)
 
+@then("the command exit code is {values}")
+def step_the_command_exit_code_is(ctx, values):
+    """
+    Compares the exit code of the previous command with a given value or comma separated list of values or ranges, e.g. '1,3,100-102'.
+    """
+    codes = []
+    parts = values.split(',')
+    for part in parts:
+        if '-' in part:  # range a-b
+            (lower, upper) = part.split('-', 1)
+            lower = int(lower.strip())
+            upper = int(upper.strip())
+            codes.extend(range(lower, upper+1))
```

Comment:
I think it will not work when upper is lower than lower and it will just loop forever